sql/step_04_all_following_steps.sql

```diff
@@ -67,5 +67,7 @@ source step_70_increase_examination_note_field.sql;
 source step_71_drop_opd_date_vis_field.sql;
 source step_72_hospital_changes.sql;
 source step_73_ward_visit_duration.sql;
+source step_72_drop_pat_deleted.sql;
+source step_73_drop_adm_deleted.sql;
```

Comment:
By using step 72 and 73 you are replacing those that already exist there: `step_72_hospital_changes` and `step_73_ward_visit_duration`; please use other numbers.